terraform/application/application.tf

```diff
@@ -11,13 +11,14 @@ module "application_configuration" {
   # Delete for non rails apps
   is_rails_application = true
 
-  config_variables = {
+  config_variables = merge(
+    var.app_env_values,
+    {
     ENVIRONMENT_NAME = var.environment
     PGSSLMODE        = local.postgres_ssl_mode
-    SIGN_IN_METHOD   = var.sign_in_method
     CLAIMS_HOST      = local.claims_host
```

Comment:
CLAIMS_HOST and PLACEMENTS_HOST can be removed from here as they are in the yaml file